core_backend/main.py

```diff
@@ -2,10 +2,19 @@
 
 import uvicorn
 from app import create_app
+from app.configs.app_config import BACKEND_ROOT_PATH
 from fastapi.logger import logger
+from uvicorn.workers import UvicornWorker
 
 app = create_app()
 
+
+class Worker(UvicornWorker):
+    """Custom worker class to allow root_path to be passed to Uvicorn"""
+
+    CONFIG_KWARGS = {"root_path": BACKEND_ROOT_PATH}
```

Comment:
All this is needed to make sure that Swagger UI works when using nginx (at `/api/docs`) and when using local dev setup (at `/docs`)